sdk/monitor/monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.30 (2025-04-09)
 
+### Bugs Fixed
+
+- Fixed process time normalized calculation returning NaN.
```

Comment:
Is there a existing bug created somewhere we can reference here?

Reply:
There isn't as we caught this before any customers discovered it